src/clients/services_proposal.rb

```diff
@@ -328,7 +328,7 @@ def open_firewall_ports plugins
       end
 
       def protected_service? service_name
-        return true if Linuxrc.vnc    && service_name == "xinetd"
+        return true if Linuxrc.vnc    && service_name == "xvnc.socket"
```

Comment:
this won't work as it uses `Service` module which works only with services and not sockets. In general as this API does not allow to disable socket, we should just remove this call.